### PR TITLE
Update Solo Score table structure

### PR DESCRIPTION
**Pull request by nanaya**
This drops existing data ( ﾟ ヮﾟ)a

### CODE REVIEW

File: app/Models/Solo/Score.php [hunk 1]
```diff
@@ -39,45 +31,77 @@ class Score extends Model
...
-        return $this->ended_at !== null;
-    }
-
-    public function complete(array $params): void
-    {
-        $this->fill($params);
+        $score = new static([
+            'beatmap_id' => $params['beatmap_id'],
+            'ruleset_id' => $params['ruleset_id'],
+            'user_id' => $params['user_id'],
```
Comment (notbakaneko) [inline app/Models/Solo/Score.php:45]: need to handle missing field or set default

**Review by notbakaneko — CHANGES_REQUESTED** [review]

File: app/Models/Solo/Score.php (hunk 1 above)
Comment (notbakaneko) [reply app/Models/Solo/Score.php:45]: I guess they're in the ScoreToken 🤔

**Review by notbakaneko — APPROVED** [review]

**Comment by nanaya** [comment]
@peppy please make sure to adjust the tables (migration entry, manual data migration etc) before deploying this.

**Comment by peppy** [comment]
yep, saw this

**Pull request merged**